src/Microsoft.Health.SqlServer/Features/Schema/Manager/ISchemaManagerDataStore.cs

```diff
@@ -15,8 +15,9 @@ public interface ISchemaManagerDataStore : IReadOnlySchemaManagerDataStore
         /// </summary>
         /// <param name="script">The script to execute</param>
         /// <param name="version">The version to update its status</param>
+        /// <param name="applyFullSchemaSnapshot">A flag to know if its a full schema snapshot</param>
```

Comment:
nit: may be something like - A flag to identify if the applying script is snapshot script or diff script

Reply:
Updated